src/controllerbluetooth.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+# This bash file connects and disconnects the Xbox One S controller to the
+# Raspberry Pis depending on the argument passed using the controller's.
+# MAC address. If the argument passed is "connect" then the controller is
+# connected, if "disconnect" is passed then the controller is disconnected
+
+option=$1
```

Comment:
Can we also change this to something more descriptive? Especially if we add another parameter for the BTADDR. Something like
`command=$1`

Reply:
And add another variable for the bluetooth address like
`address=$2`